bundles/binding/org.openhab.binding.caldav-command/README.md

```diff
@@ -177,60 +189,64 @@ In every case, the binding org.openhab.io.caldav-<version>.jar is needed. For ex
 
 openhab.cfg
 
-    ################################ CalDav Binding #######################################
-    #
-    #caldavio:<calendar-id>:url=
-    #caldavio:<calendar-id>:username=
-    #caldavio:<calendar-id>:password=
-    #caldavio:<calendar-id>:reloadInterval=<minutes>
-    #caldavio:<calendar-id>:preloadTime=<minutes>
-    #caldavio:timeZone=<e. g. Europe/Berlin>
-
-    # Dienstlicher/privater Kalender
-    caldavio:dienstlich:url=http://192.168.2.5/owncloud/remote.php/caldav/calendars/user/pers%C3%B6nlich
-    caldavio:dienstlich:username=user
-    caldavio:dienstlich:password=password
-    caldavio:dienstlich:reloadInterval=60
-    caldavio:dienstlich:preloadTime=2880
-    caldavio:timeZone=Europe/Berlin
-
-    # Müllkalender
-    caldavio:muell:url=http://192.168.2.5/owncloud/remote.php/caldav/calendars/user/m%C3%BCll
-    caldavio:muell:username=user
-    caldavio:muell:password=password
-    caldavio:muell:reloadInterval=1440
-    caldavio:muell:preloadTime=2880
-    caldavio:timeZone=Europe/Berlin
-
-    # Command-kalender``
-    caldavio:command:url=http://192.168.2.5/owncloud/remote.php/caldav/calendars/user/command
-    caldavio:command:username=user
-    caldavio:command:password=password
-    caldavio:command:reloadInterval=10
-    caldavio:command:preloadTime=1440
-    caldavio:timeZone=Europe/Berlin
-
-    # Additionally needed binding: org.openhab.binding.caldav-command-<version>.jar
-    # used to execute commands by a triggered event
-    # multiple calendars (calerdar-id) can be seperated by commas
-    #caldavCommand:readCalendars=<calendar-id>
-    caldavCommand:readCalendars=command
-
-    # Additionally needed binding: org.openhab.binding.caldav-personal-<version>.jar
-    # used to record and simulate presence and to show upcoming/active events
-    # multiple calendars (calerdar-id) can be seperated by commas
-    #caldavPersonal:usedCalendars=<calendar-id>
-    caldavPersonal:usedCalendars=dienstlich,muell
-
-    # If one of these identifiers can be found inside the place of the event, 
-    # this event will not be used for presence
-    #caldavPersonal:homeIdentifiers=<values seperated by commas>
+```
+################################ CalDav Binding #######################################
+#
+#caldavio:<calendar-id>:url=
+#caldavio:<calendar-id>:username=
+#caldavio:<calendar-id>:password=
+#caldavio:<calendar-id>:reloadInterval=<minutes>
+#caldavio:<calendar-id>:preloadTime=<minutes>
+#caldavio:timeZone=<e. g. Europe/Berlin>
```

Comment:
Let's remove the `<e. g. Europe/Berlin>` since it's repeated many times below

Reply:
This is apparently just a copy from the configuration file. I think we should just leave it in there.

Reply:
Good point. In that case we should remove the entire block.